src/eigensolver/backtransformation/mc.cpp

```diff
@@ -0,0 +1,14 @@
+#include "dlaf/eigensolver/backtransformation/mc.h"
```

Comment:
There is no license header in none of the `src/.../.../mc.cpp`. Should I add to them all?

Reply:
ah, sorry. I didn't notice that, I've just seen a new file without license header and I was triggered.

From `https://github.com/eth-cscs/DLA-Future/blob/master/misc/coding.md#files` I would say that all files should contain the license.

@rasolca what do you think about this? just to have a feedback, so that in case we want to change the current status, I will open a separate PR (no need to address this here)

Reply:
Missing license should be fixed in another PR.